File: Src/java/engine/src/main/java/org/opencds/cqf/cql/engine/elm/visiting/AliasRefEvaluator.java
package org.opencds.cqf.cql.engine.elm.visiting;

import org.opencds.cqf.cql.engine.execution.State;

public class AliasRefEvaluator {
    public static Object internalEvaluate(String name, State state) {
        return state.resolveAlias(name);
    }
}

File: Src/java/engine/src/commonMain/kotlin/org/opencds/cqf/cql/engine/elm/executing/AliasRefEvaluator.kt
package org.opencds.cqf.cql.engine.elm.executing

import kotlin.jvm.JvmStatic
import org.opencds.cqf.cql.engine.execution.State
import org.opencds.cqf.cql.engine.runtime.CqlType

object AliasRefEvaluator {
    @JvmStatic
    fun internalEvaluate(name: kotlin.String?, state: State?): CqlType? {
        return state!!.resolveAlias(name)
    }
}
